src/GameOfLife.test.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {cleanup, fireEvent, render} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import GameOfLife from './GameOfLife';
+
+afterEach(cleanup);
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<GameOfLife/>, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+it('renders input dimensions as a default state', () => {
+  const {getByText} = render(<GameOfLife/>);
+
+  expect(getByText('Zadej počet řádků:')).toBeInTheDocument();
+  expect(getByText('Zadej počet sloupců:')).toBeInTheDocument();
+});
+
+it('renders input form when input dimensions are specified and submitted', () => {
+  const {getByText, getByLabelText, getAllByLabelText} = render(<GameOfLife/>);
+
+  expect(getByText('Zadej počet řádků:')).toBeInTheDocument();
+  expect(getByText('Zadej počet sloupců:')).toBeInTheDocument();
+
+  fireEvent.change(getByLabelText('input-dimension-rows'), {target: { value: 3}});
+  fireEvent.change(getByLabelText('input-dimension-cells'), {target: { value: 2}});
+  fireEvent.click(getByText('Vygeneruj herní plán'));
+
+  expect(getAllByLabelText('input-cell')[0]).toBeVisible();
```

Comment:
Not a big deal but `getAllByLabelText('input-cell')` could be assigned to a variable so you would not have to call this function six times.